lib/stack/branch/index.js

```diff
@@ -77,6 +103,24 @@ export function Branch (http, data = {}) {
      * .then((collection) => { console.log(collection) })
      */
     this.query = query({ http, wrapperCollection: BranchCollection })
+
+    this.merge = async (mergeObj) => {
+      const url = '/stacks/branches/merge';
+      const header = { 
+        headers: { ...cloneDeep(this.stackHeaders)},
+      }
+      try {
+        const response = await http.post(url, mergeObj, header);
+        if (response.data) return response.data;
```

Comment:
@nadeem-cs if the data is not present in response, then it should throw an error with response object.